### PR TITLE
Add SIG CLI Spotlight article

### PR DESCRIPTION
**Pull request by fsmunoz**
SIG CLI Spotlight article.

Initial review is done in the contributor-site: https://github.com/kubernetes/contributor-site/pull/411

The final version there will be copied here, with any necessary adjustments, like the date.

### CODE REVIEW

File: content/en/blog/_posts/2023-07-13-sig-cli-spotlight.md
Comment (sftim) [inline content/en/blog/_posts/2023-07-13-sig-cli-spotlight.md:1]: Delete this file.

**Review by sftim — CHANGES_REQUESTED** [review]
Thanks. With a couple of tweaks, this'll be good to go.

/hold
Let's merge this once the other PR has merged.
(we can unhold the contributor site PR once this has earned LGTM and approval)

File: content/en/blog/_posts/2023-07-20-sig-cli-spotlight.md
Comment (fsmunoz) [reply content/en/blog/_posts/2023-07-20-sig-cli-spotlight.md:8]: Done in latest commit.

File: content/en/blog/_posts/2023-07-20-sig-cli-spotlight.md
Comment (sftim) [inline content/en/blog/_posts/2023-07-20-sig-cli-spotlight.md:6]:
```suggestion
```

File: content/en/blog/_posts/2023-07-20-sig-cli-spotlight.md
Comment (sftim) [inline content/en/blog/_posts/2023-07-20-sig-cli-spotlight.md:6]: D'oh
(nice to get this in after merge)
```suggestion
canonicalUrl: https://www.kubernetes.dev/blog/2023/07/13/sig-cli-spotlight-2023/
---
```

```

**Review by sftim — COMMENTED** [review]
/hold cancel

File: content/en/blog/_posts/2023-07-20-sig-cli-spotlight.md
Comment (fsmunoz) [reply content/en/blog/_posts/2023-07-20-sig-cli-spotlight.md:6]: Added in a new PR!

**Pull request merged**
